pkgs/misc/drivers/infnoise/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchFromGitHub, git, libftdi }:
+
+stdenv.mkDerivation rec {
+  name = "infnoise";
+
+  src = fetchFromGitHub {
+    owner = "13-37-org";
```

Comment:
Why are we not using https://github.com/waywardgeek/infnoise ?

Reply:
I believe waywardgeek started this project, and  13-37-org commercialized it. Looking at releases and commits, it seems that 13-37-org is the current maintainer of the project. Debian is also using the 13-37-org repo (haven't checked other distro's).